### PR TITLE
OAuth sample fix

### PR DESCRIPTION
**Pull request by ZackAllen**
* Updated readmes for the oauth sample
* Added comment to clarify limitations of UWP workflow

### CODE REVIEW

File: src/Android/Xamarin.Android/Samples/Security/OAuth/readme.md [hunk 1]
```diff
@@ -4,22 +4,40 @@ This sample demonstrates how to authenticate with ArcGIS Online (or your own por
...
+
+When you run the sample, the app will load a web map which contains premium content. You will be challenged for an ArcGIS Online login to view the private layers. Enter a user name and password for an ArcGIS Online named user account (such as your ArcGIS for Developers account). If you authenticate successfully, the traffic layer will display, otherwise the map will contain only the public basemap layer.
 
 ## How it works
 
-1. When the app loads, a web map containing premium content (world traffic service) is loaded in the map view.
-2. In response to the attempt to access secured content, the `AuthenticationManager` shows an OAuth authentication dialog from ArcGIS Online.
-3. If the user authenticates successfully, the world traffic service appears in the map. Otherwise, only the basemap appears.
+1. Set the `AuthenticationManager`'s `AuthenticationChallengeHandler` to a `DefaultAuthenticationChallengeHandler`.
+2. Create an `OAuthConfiguration` specifying the portal URL, client ID, and redirect URL.
```
[the remark this answers is not in the document]
Comment (ZackAllen) [reply src/Android/Xamarin.Android/Samples/Security/OAuth/readme.md]: We link to this doc in the readme - https://developers.arcgis.com/documentation/core-concepts/security-and-authentication/mobile-and-native-user-logins/ .

[the remark this answers is not in the document]
Comment (nixta) [reply src/Android/Xamarin.Android/Samples/Security/OAuth/readme.md]: Cool. That works.

File: src/Android/Xamarin.Android/Samples/Security/OAuth/readme.md [hunk 2]
```diff
@@ -20,24 +14,25 @@ When you run the sample, the app will load a web map which contains premium cont
 
 ## How it works
 
-1. Set the `AuthenticationManager`'s `AuthenticationChallengeHandler` to a `DefaultAuthenticationChallengeHandler`.
-2. Create an `OAuthConfiguration` specifying the portal URL, client ID, and redirect URL.
-3. Add the OAuth configuration to the authentication manager.
-4. Load a map with premium content requiring authentication to automatically invoke the default authentication handler.
+1. Set the `AuthenticationManager`'s `ChallengeHandler`.
```
Comment (nixta) [inline src/Android/Xamarin.Android/Samples/Security/OAuth/readme.md]: I'm guessing this is actually necessary (i.e. .NET doesn't wire up a default challenge handler automatically)?

Comment (ZackAllen) [reply src/Android/Xamarin.Android/Samples/Security/OAuth/readme.md]: Just checked, it is necessary for a proper workflow. You have to login twice without it.

Comment (nixta) [reply src/Android/Xamarin.Android/Samples/Security/OAuth/readme.md]: Ok. Thanks for checking. Looks good then. Thanks for bearing with me!

**Pull request merged**